EthOn_glossary.md

```diff
@@ -62,27 +62,18 @@ A scalar value equal to the number of ancestor Blocks. The genesis Block has a n
 [`ethon:blockSize`](http://ethon.consensys.net/blockSize)   
 The size of the the Block header in RLP format in bytes.
 ## C
-### Call Contract Message
-[`ethon:CallContractMsg`](http://ethon.consensys.net/CallContractMsg)   
-A Call Contract Message is a Contract Message that calls a function in another Contract.
-### Call Transaction
-[`ethon:CallTx`](http://ethon.consensys.net/CallTx)   
-A type of Transaction that is directed towards a Contract Account and calls a method in the Contract's code.
+### Call Message
+[`ethon:CallMsg`](http://ethon.consensys.net/CallMsg)   
+A type of Message that is directed towards a Contract Account and calls a method in the Contract's code.
 ### Contract Account
 [`ethon:ContractAccount`](http://ethon.consensys.net/ContractAccount)   
 A notional object existent only within the hypothetical state of Ethereum. Has an intrinsic address and thus an associated Account; the Account will have non-empty associated EVM Code. Incorporated only as the Storage State of that Account.
 ### Contract Message
 [`ethon:ContractMsg`](http://ethon.consensys.net/ContractMsg)   
 A Contract Message is passed between a Contract Account and any other Account (External or Contract). It is the result of an execution chain originally triggered by an External Account.
-### Create Contract Message
-[`ethon:CreateContractMsg`](http://ethon.consensys.net/CreateContractMsg)   
-A Create Contract Message is a subtype of a Contract Message that results in creation of a new Contract Account.
-### Create Transaction
```

Comment:
It was changed to `Create Message` as part of @sunnya97's pull request #91 `message types`, which simplified things by grouping `Transaction`, `Create Message`, `Call Message` and `Contract Message` under the `Message` class.